pages/docs/couple-your-code/couple-your-code-just-in-time-mapping.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Just-in-time data mapping
+permalink: couple-your-code-just-in-time-mapping.html
+keywords: api, adapter, mapping, meshes, just-in-time
+summary: "You can read and write data by providing dynamic coordinates instead of static vertex IDs using specific optional API functions."
+---
+
+{% version 3.2.0 %}
+This feature is new in preCICE version 3.2.0.
+{% endversion %}
+
+{% experimental %}
+This is an experimental feature.
+{% endexperimental %}
```

Comment:
We should add somewhere (here?) that the experimental switch in the config needs to be enabled, right?